feat: 마일스톤 요청 목업 API

hiro032 opened the pull request:
마일스톤 목록 조회 API를 작성했습니다

hiro032 committed:
feat: 마일스톤 요청 목업 API

Dae-Hwa commented on be/issue-tracker-be/src/main/java/com/codesquad/issuetracker/milestone/MileStoneDummyData.java:
```diff
@@ -0,0 +1,25 @@
+                    .name("milestone명")
+                    .description("milestone설명")
+                    .dueDate(LocalDate.of(2020, 12, 25))
+                    .isClosed(true)
```
목업에 isClosed로 만드셨으면 반대되는 예시도 하나 넣어주면 좋을 것 같아요
이유는 클라이언트에서 목업을 받았을 때 open상태여야 화면에 출력될테니까요

hiro032 committed:
fix: MileStoneDummyData.MileStoneResponse에서 false를 반환하는 더미데이터 추가

hiro032 committed:
fix: IssueDummyData.issueDetailResponse와 IssueDummyData.issueResponses에서 사용하던 milestoneMockup()을 MileStoneDummyData.MileStoneResponse()로 대

Dae-Hwa commented on be/issue-tracker-be/src/main/java/com/codesquad/issuetracker/milestone/MileStoneDummyData.java:
```diff
@@ -0,0 +1,47 @@
+    public static Set<MileStoneResponse> MileStonesResponse() {
+        return new HashSet<>(Arrays.asList(
+                MileStoneResponse.builder()
+                    .id(1L)
+                    .name("milestone명")
+                    .description("milestone설명")
+                    .dueDate(LocalDate.of(2020, 12, 25))
+                    .isClosed(true)
+                    .openedIssueCount(1)
+                    .closedIssueCount(1)
+                    .build(),
+
+                MileStoneResponse.builder()
+                    .id(2L)
+                    .name("milestone명")
+                    .description("milestone설명")
+                    .dueDate(LocalDate.of(2020, 12, 23))
+                    .isClosed(false)
+                    .openedIssueCount(2)
+                    .closedIssueCount(2)
+                    .build()
```
아래에 메소드를 잘 작성하면 여기도 대체할 수 있을 것 같습니다

Dae-Hwa commented on be/issue-tracker-be/src/main/java/com/codesquad/issuetracker/milestone/MileStoneDummyData.java:
```diff
@@ -0,0 +1,47 @@
+
+    public static MileStoneResponse MileStoneResponse() {
+        return  MileStoneResponse.builder()
+                    .id(1L)
+                    .name("milestone명")
+                    .description("milestone설명")
+                    .dueDate(LocalDate.of(2020, 12, 25))
+                    .isClosed(false)
+                    .openedIssueCount(1)
+                    .closedIssueCount(1)
+                    .build();
```
의미있게 작성하면 위의 문제도 함께 해결할 수 있을 것 같습니다.

hiro032 committed:
fix: MileStoneDummyData수정

MileStoneDummyData.mileStonesResponse에서 closeMileStoneResponse, openMileStoneResponse를 사용하도록 수정.
MileStoneDummyData.mileStonesResponse 메서드 네이밍을 자바 컨벤션에 맞게 수정

hiro032 committed:
Merge branch 'be/mockup/main' into be/mockup/milestones

Dae-Hwa commented on be/issue-tracker-be/src/main/java/com/codesquad/issuetracker/milestone/MileStoneDummyData.java:
```diff
@@ -0,0 +1,42 @@
+
+    public static MileStoneResponse openMileStoneResponse() {
+        return  MileStoneResponse.builder()
+                    .id(1L)
```
id가 중복되네요!

hiro032 committed:
Merge branch 'be/mockup/main' into be/mockup/milestones

hiro032 committed:
fix: IssueDummyData충돌 해결

MileStoneDummyData.closeMileStoneResponse의 중복 id값 수정
